sqltest/sql_test.go

```diff
@@ -4,29 +4,38 @@
 package sqltest
 
 import (
-	log "github.com/sirupsen/logrus"
-	"github.com/squareup/pranadb/internal/testcerts"
 	"io/ioutil"
 	"os"
 	"testing"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/squareup/pranadb/internal/testcerts"
 )
 
 func TestSQLFakeCluster(t *testing.T) {
 	log.Debug("Running TestSQLFakeCluster")
-	testSQL(t, true, 1, 0, false, tlsKeysInfo)
+	testSQL(t, true, 1, 0, false, false, tlsKeysInfo)
 }
 
 func TestSQLFakeClusterUsingHTTPAPI(t *testing.T) {
 	log.Debug("Running TestSQLFakeClusterUsingHTTPAPI")
-	testSQL(t, true, 1, 0, true, tlsKeysInfo)
+	testSQL(t, true, 1, 0, true, false, tlsKeysInfo)
 }
 
 func TestSQLClusteredThreeNodes(t *testing.T) {
 	if testing.Short() {
 		t.Skip("-short: skipped")
 	}
 	log.Info("Running TestSQLClusteredThreeNodes")
-	testSQL(t, false, 3, 3, false, tlsKeysInfo)
+	testSQL(t, false, 3, 3, false, false, tlsKeysInfo)
+}
+
+func TestSQLClusteredThreeNodesTLS(t *testing.T) {
```

Comment:
Best to move this into it's own go file and exclude it with a build flag, like we do with large_cluster_tests.go. We don't want the standard test suite taking too long to run and we can run the mTLS tests with the build flag once our EC2 runner is working.

Reply:
Done